documents/PropBetPredictProposal.md

```diff
@@ -0,0 +1,145 @@
+## Proposal of MVP version of Bot Blitzer
+
+### Background
+
+We are likely to lose motivation on this project.  I think the best way for this project to stay interesting, is if we can use some form of what we are building almost immediately.  What version of this can be put into immediate use by us with very little upfront engineering work?  The purpose of this doc is to describe that.
+
+### Approach
+
+We focus on NBA because the NBA has games daily until June.  
```

Comment:
... I can think of another sport that also has games until June...

Reply:
Cricket?